src/controllers/message.js

```diff
@@ -17,4 +17,32 @@ const createMessage = async (req, res) => {
     res.status(StatusCodes.CREATED).json({ message })
 }
 
-module.exports = { createMessage }
+// Mark conversation as read
+const markConversationAsRead = async (req, res) => {
+    try {
+        const { userId } = req.user
+        const { postedByUser, messageRead } = req.body
+        // update multiple message at the same time
+        await Message.updateMany(
+            {
+                postedByUser,
+                receivedByUser: userId,
+            },
+            req.body,
+            { new: true, runValidators: true }
+        )
+        // returning all updated message with new messageRead status
+        const updatedMessages = await Message.find({
+            postedByUser,
+            receivedByUser: userId,
+        })
```

Comment:
Can you return the entire conversation here? This would be the same as BACK-48 in this case.

Reply:
@jpremo I've updated my code, could you please take a look and approve it if everything's ok? Thank you.